gremlin-go/driver/connection_test.go

```diff
@@ -258,21 +257,21 @@ func TestConnection(t *testing.T) {
 
 	t.Run("Test createConnection without valid server", func(t *testing.T) {
 		connection, err := createConnection(invalidHostValidPortValidPath, newLogHandler(&defaultLogger{}, Info,
-		language.English), testNoAuthAuthInfo, testNoAuthTlsConfig, keepAliveIntervalDefault, writeDeadlineDefault)
+			language.English), testNoAuthAuthInfo, testNoAuthTlsConfig, keepAliveIntervalDefault, writeDeadlineDefault)
```

Comment:
This is kind of a weird formatting. Was this done by `go fmt`?

Reply:
yeah that was an auto format